package.json

```diff
@@ -10,14 +10,13 @@
     "clean": "./scripts/bash/clean.sh",
     "build:bootstrap": "./scripts/bash/bootstrapBuild.sh",
     "build:dictionary": "node scripts/node/dictionary/build.js",
-    "build:styles": "yarn styles:min && yarn styles:full && yarn styles:docs",
+    "build:styles": "yarn styles:min && yarn styles:full",
     "build:icons": "yarn icons:optimize && yarn icons:react && yarn icons:transpile && yarn icons:css && yarn icons:docs",
     "build:comp": "yarn comp:css && yarn comp:jsx && yarn comp:docs",
     "build:distCSS": "NODE_ENV=production postcss src/assets/all-styles.css -o ./lib/assets/all-styles.min.css --config=./postcss.config.js",
     "build:full": "yarn clean && yarn build:bootstrap && yarn build:dictionary && yarn build:styles && yarn build:icons && yarn build:comp && yarn build:distCSS",
     "styles:min": "NODE_ENV=production postcss src/base-styles/index.css --verbose --output=lib/base-styles/base-styles.min.css --config=./postcss.config.js",
     "styles:full": "postcss src/base-styles/index.css --verbose --output=lib/base-styles/base-styles.full.css --config=./postcss.config.js",
-    "styles:docs": "NODE_ENV=base_styles_docs postcss ./lib/base-styles/base-styles.full.css --output=/dev/null",
```

Comment:
This should also clean up the postcss config.

`./postcss.config.js` contains some logic that checks the node env for `base_styles_docs` and conditionally adds plugins from `postcss.mdcss.js`